frontend/src/metabase/containers/Overworld.jsx

```diff
@@ -269,6 +274,56 @@ class Overworld extends React.Component {
   }
 }
 
+const PIN_MESSAGE_STORAGE_KEY = "mb-admin-homepage-pin-propaganda-hidden";
+
+class AdminPinMessage extends React.Component {
+  state = {
```

Comment:
I think this needs to go in the `constructor` function otherwise it just gets resolved once when the app starts. If the admin dismisses, navigates to another page, and back it will probably still show up. 

If you verify that's not the case go ahead and merge, otherwise just wrap it in the usual `constructor(props) { super(props); ... }`

Reply:
@tlrobinson Yeah I had wondered about that too. I tested it with and without using a constructor method and it didn't seem to affect it so I figured I'd go with the simple state initialization.